content/_ext/yaml_gallery_generator.py

```diff
@@ -62,7 +143,10 @@ def build_from_yaml(filename, display_name):
     panels_body = '\n'.join(panels_body)
 
     panels = f"""
-# {display_name} Gallery
+# {display_name}
+
+{menu_html}
+
 ````{{panels}}
 :container: full-width
```

Comment:
@jukent, removing this setting fixes the container overflow

```suggestion
```


![Screenshot 2021-05-13 163609](https://user-images.githubusercontent.com/13301940/118196005-7f659080-b409-11eb-9cc3-68145fffd16f.png)

Reply:
Thanks! I saw that it was set to `!important` in the CSS and wasn't sure if it looked better when removed. I think ideally since there is no need for a right nav bar on the gallery, the top nav bar content would expand to take up that empty space, as would the gallery. This seems to do that.

Reply:
>  This doesn't do that -- but I might change the appearance of the cards anyway and will worry about this stylistic preference later.

Sounds good..